oauthenticator/generic.py

```diff
@@ -134,6 +135,7 @@ def authenticate(self, handler, data=None):
             'name': resp_json.get(self.username_key),
             'auth_state': {
                 'access_token': access_token,
+                'refresh_token': refresh_token,
```

Comment:
do you prefer the key to always be present (like this) but with `None` as value or only put the key in if the value is not `None`? I am easy either way.

Reply:
No preference from me